blocks/contact-form/contact-form.js

```diff
@@ -69,12 +70,24 @@ export default function decorate(block) {
     }
   });
   // Specify submitContactInfo arguments
-  const submitInfo = (event) => {
+  const submitInfo = async (event) => {
     // To prevent the button from scrolling up when clicked
     if (event) {
       event.preventDefault();
     }
-    submitContactInfo(formInputs);
+
+    // Make sure no inputs are empty
+    const emptyInputs = [...formInputs].filter(
+      (input) => input.required && !input.value.trim(),
+    );
+
+    if (emptyInputs.length > 0) {
+      emptyInputs.forEach((input) => input.classList.add('required-empty-input'));
+      return;
+    }
+
+    await submitContactInfo(formInputs);
```

Comment:
what happens once you successfully submit? or what happens when you get an error? you `throw` an error if the request doesn't go through, but it is not handled anywhere, or is it?